### PR TITLE
Fix: save views

### PR DESCRIPTION
**Pull request by gcofficial**

### CODE REVIEW

**Comment by xerox-h20** [comment]
Setting the saved view doesn't work if widgets are already on the desktop. Scenario example:
1. Select 'Trading' in Widgets dropdown. 2. Make changes to positioning of tiles. 3. Save current View. 4. Select 'Research' to remove all widgets. 5. Select 'Trading' again to make widgets reappear. 6. When clicking on the new saved View the widgets won't move.

**Comment by gcofficial** [comment]
Fixed.

File: src/components/grid/Tile.vue [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
-      style: {
+  computed: {
+    style() {
+      return {
         left: this.left + 'px',
         top: this.top + 'px',
-        zIndex: 0,
-      },
-    };
+        zIndex: this.getMaxZIndex(),
```
Comment (xerox-h20) [inline src/components/grid/Tile.vue]: This causes an incorrect display stack order. E.g. Widget 1 and Widget 2 are displayed with z-index = 1 and 2 respectively. Dragging Widget 1 over Widget 2 will not make it go "under" Widget 2.

**Comment by gcofficial** [comment]
> Setting the saved view doesn't work if widgets are already on the desktop. Scenario example:
> Select 'Trading' in Widgets dropdown. 2. Make changes to positioning of tiles. 3. Save current View. 4. Select 'Research' to remove all widgets. 5. Select 'Trading' again to make widgets reappear. 6. When clicking on the new saved View the widgets won't move.

Fixed

**Pull request merged**